src/test/scala/com/mongodb/spark/rdd/partitioner/MongoFixedNumberPartitionerSpec.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2016 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.spark.rdd.partitioner
+
+import org.bson._
+import com.mongodb.spark._
+
+class MongoFixedNumberPartitionerSpec extends RequiresMongoDB {
+
+  private val partitionKey = "_id"
+  private val pipeline = Array.empty[BsonDocument]
+
+  "MongoFixedNumberPartitioner" should "partition the database as expected" in withSparkContext() { sc =>
+    sc.parallelize((0 to 1000).map(i => BsonDocument.parse(s"{ _id: $i }"))).saveToMongoDB()
+
+    val rightHandBoundaries = (0 to 1000 by 100).map(x => new BsonInt32(x))
+    val locations = PartitionerHelper.locations(MongoConnector(sparkConf))
+    val expectedPartitions = PartitionerHelper.createPartitions(partitionKey, rightHandBoundaries, locations)
```

Comment:
Since the partitioner tests are using PartitionerHelper.createPartitions to create the expected partitions, better have a direct test of PartitionerHelper.createPartitions itself.